src/Tickets/Commerce/Gateways/Stripe/Hooks.php

```diff
@@ -114,7 +115,7 @@ public function include_payment_buttons( $file, $name, $template ) {
 	 */
 	public function maybe_create_stripe_payment_intent() {
 
-		if ( ! tribe( Merchant::class )->is_connected() ) {
+		if ( ! tribe( Merchant::class )->is_connected() && tribe( Module::class )->is_checkout_page() ) {
```

Comment:
We should probably create the order here and the payment intent from the cart and avoid refreshing it.

Reply:
See my comment above. To avoid refactoring the Orders, we're creating the payment intent here and then retrieving the same PI from the Stripe API when creating the Order.